inventory/byo/hosts.ose.example

```diff
@@ -22,6 +22,7 @@ ansible_ssh_user=root
 # Debug level for all OpenShift components (Defaults to 2)
 debug_level=2
 
+#--------# REVIEW: what are the *current* valid values? #--------#
 # deployment type valid values are origin, online, atomic-enterprise, and openshift-enterprise
```

Comment:
`valid deployment types for new installations are origin and openshift-enterprise.`